### DISCUSSION TITLE
UG DQ 02 — Top-Down Analysis and Ontological Commitments

### DISCUSSION THREAD

**Discussion by johnbeve** [Applied-Ontology-Education/Ontology-and-Intel-Analysis-Fall-2024#2] [Undergraduate Discussion Questions]
**GitHub Discussion title:** `UG DQ 02 — Top-Down Analysis and Ontological Commitments`

**Maximum length:** 500 words

Complete the Week 2 readings and lectures before responding.

## Prompt

Choose an intelligence-analysis concept that appears important in policy, doctrine, analytic methodology, or professional practice.

Examples might include *threat*, *capability*, *intent*, *requirement*, *indicator*, *source*, or *assessment*.

Answer the following:

1. How is the concept characterized in the source material you are using?
2. What assumptions or ontological commitments are built into that characterization?
3. What entities would need to exist for the characterization to be correct?
4. What could go wrong if an ontology copied the terminology without examining those commitments?

Conclude with a short proposed definition or modeling principle.

**Comment by drewbonge12** [comment]
Within intelligence-analysis, one important concept that can be seen is patterns. Patterns are known as a meaningful repetition of the arrangements of things. These things can span from art in so far as the way shapes can be regularly repeated, all the way to behaviors such as the order in which you put on your clothes in the morning from day-to-day. Mainly, the greatest characterization that you can see from a pattern in intelligence-analysis is the identification of useful trends seen across observations. 
The ontological commitments that are built into the characterization of patterns stem from the idea that there has to be things in the world that can be observed, and that they can be arranged in any way. For example, your socks, underwear, pants, shirt, and shoes are all things that exist in this world; however, they are no good in presenting a pattern without the idea that they can also be arranged, or, in my example, the order in which you put them on. Another useful ontological commitment can be seen in that the arrangement itself can constitute something of meaning. The clothing itself bears no pattern, instead the relationship between clothing and the order they are put on day-to-day creates the pattern. 
For the characterization to be correct, there would need to be observation of entities, arrangements between them, and an identifiable way that an arrangement or repetition is meaningful. 
The main thing that can go wrong if an ontology copied the terminology without examining these listed commitments is that an analyst may treat all arrangements and repetitions as patterns, thus treating coincidence as a non-possibility. If this were the case, analysts would identify patterns in spots where a pattern does not actually exist. For example, in 1990 there was a weird 1:1 ‘pattern’ seen in the amount of people that drowned in swimming pools, and the amount of movies Nicholas Cage appeared in; That is not a pattern in the way we define meaningful relationships and arrangements, that is just a concerning coincidence. If an analyst assumed all arrangements and their repetitions could be patterns, it would result in the confusion of simple observations or coincidences with meaningful relationships between observations.
A short proposed modeling principle could be that a pattern should be characterized by meaningful arrangements or relationships that can support identifiable similarities and connections over a series of observations.

**Comment by LaibahA** [comment]
One concept that appears within intelligence-analysis is "source". A source is typically from who, what, and/or where something comes from. Intelligence analysts work with lots of information and data that may come from various sources which can be different people, situations, repositories, etc. They're able to procure that information from the _source_, which tells us that what's important isn't what the information is (knowledge, data, pictures, audio, etc), but more importantly, about providing the information that analysts are able to use to make assessments and assertions about reality. Being able to characterize a source is vital because analysts need to know what produced/provided information and how that source is related to the information. For example, a known person may be a more reliable source than a tip off, and a database may be more consistent than a person. The main characterization is that it's something that provides information that intelligence analysts use for their assessments. 

The ontological commitments of a source are that there must be some information, there is something that provides the information, and that there's some relationship between the creator/provider and the information being provided. The entities required for this characterization are a source, an information entity, and an explicit relationship connecting the source to the information it provides. Sometimes the same or similar information comes from different sources, which can be useful if an analyst wants to cross check something. But, if all the sources are connected to the same information entity, it can be hard to trace back to make assessments such as which source was the most accurate, or which one gave the earliest information. Keeping the source explicitly linked to one source helps maintain the lineage of the information. 

The two entities must exist and there must be an established relationship between them in order for the source to be assessed accurately. If an ontology copied the terminology without these commitments, there may be inconsistency of where information is really coming from. The same information can be represented in different ways, like the original way the information was received from the actual source, then a report about the information, to a presentation about the report of the information. The latter points aren't the source, they don't provide the original information, they just represent the information differently because they don't have the same relationship that the original source and the information have. 

A short modeling principle for "source" in the context of intelligence-analysis could be to represent it as an entity is defined to provide information, and establish an explicit relationship between this entity and the information (another entity) that it provides.

**Comment by adriankoz-jos** [comment]
A particularly important concept in intelligence analysis is that of a *threat*.   As seen in the first week's Miller reading, the philosophical definition of intelligence is necessarily intertwined with that of security, whether it be the security of a nation-state, an individual, or an organization. Miller characterizes intelligence analysis as inherently adversarial in context: the Hoffmann/Chalati/Dogan reading from this week explores a sociological approach to intelligence analysis which provides a framework that explains the occurrence of transnational and cooperative intelligence practices. 
These definitional contradictions result in the potential for significantly different approaches and descriptions of what constitutes a threat: if, as in Miller's approach, all intelligence relations are inherently adversarial, cooperation between nation-states and entities would be sparse and filled with backstabbing and intrigue at best, and completely nonexistent at worst. The Hoffman reading's examples highlight that in reality, inter-organizational intelligence sharing and cooperation is commonplace and fits well into an ontological framework that accounts for it. This is not to say that there is not risk and threat assessment built into Miller's definition of intelligence that allows for cooperation under certain circumstances, but an ontology based entirely on the assumption that nation states and other intelligence agencies view each other suspiciously and adversarially means that allied entities would constantly be spending organizational capital on threat assessment directed at each other rather than cooperating.
A better ontological definition of *threat* would take into account inter-organizational relations, relationships between states, and various geopolitical aims (whether cooperative or adversarial) to create a functional and working definition of threat that is capable of more accurately assessing threat vectors and likely vs unlikely sources.
